### DISCUSSION TITLE
Game lobby

### DISCUSSION THREAD

**Discussion by alrusdi** [bafolts/terraforming-mars#1277] [Ideas]
Ability to create game but do not start it.
All pending games are shown on the lobby page.
Each "seat" of the game can be taken by anyone who visits the lobby page.
When "table" is full game starts automatically.
May be we need some kind of auth for it.
Would be great to have possibility to get invite link for each game in the lobby

**Comment by ssimeonoff** [comment]
This means that game will not be gathered via Discord?

**Comment by alrusdi** [comment]
> This means that game will not be gathered via Discord?

It is just an idea. It pops up in my mind every time when i see my group is trying to get players for the game

**Comment by ap3h3ad** [comment]
Some kind of lobby would be nice and could even lay the foundation for some kind of tournament, although that most definately would require some kind of authentication and accounting.
